Gemfile

```diff
@@ -52,7 +56,7 @@ gem 'wicked_pdf'
 gem 'silencer' # Silence certain Rails logs
 gem 'wkhtmltopdf-heroku'
 gem 'remotipart', '~> 1.2' # Async file uploads
-gem 'faker' # Generate fake data
+gem 'faker', '~> 1.7.3' # Generate fake data
```

Comment:
This one is also not needed

Reply:
The version suffix, `'~> 1.7.3'`? Or the Gem itself?

Reply:
Version